### DISCUSSION TITLE
Knowing hover state of perticular row

### DISCUSSION THREAD

**Discussion by Lingareddyvinitha** [TanStack/table#5198] [Q&A]
I want to implement the functionality like when I hover over any row then it should display the floating delete button. (so that when clicks on the button row would be deleted and delete button only show when I hover on the table row)

Is there any way to achieve this using react-table or any code sample of above problem?

Thanks, in advance.

**Comment by Khalifa1997** [comment]
@Lingareddyvinitha Any news on this so far?
